### DISCUSSION TITLE
Unable to reproduce results of the SFR pincell multiphysics tutorial

### DISCUSSION THREAD

**Discussion by nancysemwal** [neams-th-coe/cardinal#993] [Q&A] [answered]
I am following the [SFR pincell multiphysics tutorial](https://cardinal.cels.anl.gov/tutorials/pincell_multiphysics.html). But I am unable to reproduce the plots shown at the end of the tutorial. I am viewing the results from the `openmc_out.e` file in Paraview.

This is what my `cell_temperature` plot looks like. You can see the I'm not getting any temperature variation - 

![image](https://github.com/user-attachments/assets/d58828ea-fc5a-4f80-b697-1160b2a65de5)

These are what my `nek_temp` & `moose solid temperature` plots look like respectively. These are also wrong as seen in the plot legend (since maximum temperature is 1100 K for me while its 736.3 K in the tutorial) - 

![image](https://github.com/user-attachments/assets/fb040024-88c5-4e00-96fe-adf3e171a20e)

![image](https://github.com/user-attachments/assets/349db9b3-74a4-4d2d-8e4a-00225ea63512)

How can I diagnose whats wrong with the simulation from a Physics perspective? I am a computer science engineer so physics is not my strong suit. I have run the same commands given in the tutorial and used the same input files without changing any parameter. Also I got `finished with exit code 0` message at the end of the simulation so I understand nothing went wrong.

I am on the `devel` branch of the repository and the last commit on this was around 9 Nov, 2024 (checked using `git log`). I checked and no updates have happened to the tutorial since then.

How do I move further?

**Comment by aprilnovak** [comment, marked as answer]
Hi Nancy,

In order to view the solution on each Picard iteration, you need to toggle through the timesteps with the icon shown below. Because you are viewing time step 0, you are only seeing the initial conditions (not the actual converged solve).

<img width="386" alt="Screenshot 2024-11-29 at 8 13 20 AM" src="https://github.com/user-attachments/assets/a263e974-f1c5-42f3-9cfa-161b38a2e841">

Please let me know if this works for you!
